iOS Example Frame Carthage/iOS Example Frame Carthage.xcodeproj/project.pbxproj

```diff
@@ -23,6 +23,7 @@
 /* Begin PBXBuildFile section */
```

Comment:
@ehab-al-cko 

> Payment form
> 1. when scroll to up, the navigation bar background is transparent
> 2. summary gray line should be green
> 3. error view missing the image

1. ✅ background fixed
2. 🔴 summary gray line is not in the design and I personally do not believe it should be green. Task was to implement design, would be happy to go through design decisions with the designer :)
3. 🔴 you are assuming every error should show an image. Similar to previous comment, given the design does not show errors I think we should be prepared that consumer might want to not use image


> Billing form
> 1. All input fields are showing numeric key pad
> 2. Country field is not showing current country like United Kingdom
> 3. All errors are hidden
> 4. Optional fields are not optionals anymore

1. ✅ Thank you, missed that in reusing components. Fixed now
2. ❓ Is it supposed to do that? Only reason its been doing it so far is the existence of pre-filled data, which I've purposely left out now. Not sure how you plan to achieve this, may be worth catching up over it 🤔 
3. 🟡 The design doesn't show any errors, do we consider consumer has to provide errors for all the input fields?
4. ❓ I am not sure what optional fields are non optional anymore means. I've just tested it and everything that is optional is optional and everything that is mandatory is mandatory. Can you please detail?

Reply:
I would further dig into options being hidden on Billing form. When are they supposed to be shown as my understanding if the flow would be:
- all included - Done button enabled
- any missing - Done button disabled

I know I must be misunderstanding, but not sure where and how?

Reply:
If it's optional field then it can be empty or filled then `done` will be enabled anyway.

Reply:
The problem is the buttons enabled / disabled states were not passed accurately then. Please see [latest commit](https://github.com/checkout/frames-ios/pull/238/commits/11039c12e279bc5fd94884c29cce70f221c09977) which fixes that in the current logic.